src/Raven.Client/Documents/Changes/DatabaseChanges.cs

```diff
@@ -206,6 +212,9 @@ public IChangesObservable<OperationStatusChange> ForOperationId(long operationId
 
         public IChangesObservable<OperationStatusChange> ForAllOperations()
         {
+            if (string.IsNullOrEmpty(_nodeTag))
+                ThrowOperationsNodeTagNeeded();
```

Comment:
Here and above are the possible "breaking changes". For users who do:

```
var changes = store.Changes();

changes.ForOperationId(5);
changes.ForAllOperations();
```

@ppekrol Thoughts? 

it will start throwing since the node tag was not provided. From now the valid usage would be:

```
var changes = store.Changes("db-name", "A");

changes.ForOperationId(5);
changes.ForAllOperations();
```

Reply:
What if we mark those two as deprecated:

https://github.com/ravendb/ravendb/blob/3894f2909305381ff070a640a1240a6dfb709703/src/Raven.Client/Documents/Changes/IDatabaseChanges.cs#L26-L32

and introduce new overloads with mandatory `nodeTag` parameter?

Reply:
for 5.4 and 6.0 do not throw, for 6.1 open an issue  with breaking change tag and throw

Reply:
Let's maybe assert here in debug. so We will not do that mistake :-)

Reply:
Done